dataGen.js

```diff
@@ -0,0 +1,1204 @@
+// },
+// {
+//   "id": 123,
+//   "title": "marfa bottle blue retro",
+//   "artist": "The Chainsmokers",
+//   "genre": "Pop",
+//   "album": "Memories... Do Not Open"
+// },
+// {
+//   "id": 124,
+//   "title": "bespoke forage do Authentic",
+//   "artist": "Lindsey Stirling",
+//   "genre": "R&B",
+//   "album": "Lindsey Stirling"
+// },
+// {
+//   "id": 125,
+//   "title": "trade direct belly pork",
+//   "artist": "The Chainsmokers",
+//   "genre": "Electronic",
+//   "album": "Memories... Do Not Open"
+// },
+// {
+//   "id": 126,
+//   "title": "chillwave sriracha Gastropub snackwave",
+//   "artist": "Ed Sheeran",
+//   "genre": "HipHop",
+//   "album": "Multiply"
+// },
+// {
+//   "id": 127,
+//   "title": "juice green proident tousled",
+//   "artist": "Sam Smith",
+//   "genre": "R&B",
+//   "album": "In The Lonely Hour"
+// },
+// {
+//   "id": 128,
+//   "title": "bulb edison forage them",
+//   "artist": "Phoenix",
+//   "genre": "HipHop",
+//   "album": "Wolfgang Amadeus Phoenix"
+// },
+// {
+//   "id": 129,
+//   "title": "of heard haven't probably",
+//   "artist": "Marian Hill",
+//   "genre": "Jazz",
+//   "album": "Sway"
+// },
+// {
+//   "id": 130,
+//   "title": "you elit cronut ad",
+//   "artist": "Ed Sheeran",
+//   "genre": "Rock",
+//   "album": "Multiply"
+// },
+// {
+//   "id": 131,
+//   "title": "pug wolf gentrify iPhone",
+//   "artist": "Jasmine Thompson",
+//   "genre": "Pop",
+//   "album": "Another Bundle of Tantrums"
+// },
+// {
+//   "id": 132,
+//   "title": "Reprehenderit park echo squid",
+//   "artist": "Rick Astley",
+//   "genre": "HipHop",
+//   "album": "Whenever You Need Sombeody"
+// },
+// {
+//   "id": 133,
+//   "title": "ennui etsy sartorial adipisicing",
+//   "artist": "Michael Jackson",
+//   "genre": "HipHop",
+//   "album": "Thriller"
+// },
+// {
+//   "id": 134,
+//   "title": "crucifix non Marfa vice",
+//   "artist": "Beyonce",
+//   "genre": "Rock",
+//   "album": "I am... Sasha Fierce"
+// },
+// {
+//   "id": 135,
+//   "title": "chartreuse qui sartorial blog",
+//   "artist": "Ella Vos",
+//   "genre": "Rock",
+//   "album": "Words I Never Said"
+// },
+// {
+//   "id": 136,
+//   "title": "DIY plaid laborum cronut",
+//   "artist": "Daughter",
+//   "genre": "Pop",
+//   "album": "Wild Youth EP"
+// },
+// {
+//   "id": 137,
+//   "title": "Consequat tumblr enim culpa",
+//   "artist": "Daughter",
+//   "genre": "Country",
+//   "album": "Wild Youth EP"
+// },
+// {
+//   "id": 138,
+//   "title": "esse microdosing quinoa lorem",
+//   "artist": "Michael Jackson",
+//   "genre": "Jazz",
+//   "album": "Thriller"
+// },
+// {
+//   "id": 139,
+//   "title": "truck food bag tote",
+//   "artist": "Ed Sheeran",
+//   "genre": "Blues",
+//   "album": "Multiply"
+// },
+// {
+//   "id": 140,
+//   "title": "keffiyeh kickstarter church-key Freegan",
+//   "artist": "Rick Astley",
+//   "genre": "Pop",
+//   "album": "Whenever You Need Sombeody"
+// },
+// {
```

Comment:
Please only use songs 0-140 because any songs after id #140 have blank titles. Ringo and I believe that 140 songs should be good.

Reply:
Removed thank you!